include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
> All files in include/rdma are installed and visible. fi_log and fi_prov are usable by external providers. It is a goal to support out of tree providers, so that a single provider can be updated without re-compiling the entire library and replacing the build-in providers. I'm guessing that we can support that by building a dynamic provider in-tree and only installing the provider library, if we didn't want to allow building the provider out of tree.
> 
> I'm not aware of any providers being built out of tree.

Isn't the A3CUBE provider out of tree?